src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ThreadPoolMetricsCollector.java

```diff
@@ -52,6 +57,24 @@ public void collectMetrics(long startTime) {
 
         while (statsIterator.hasNext()) {
             Stats stats = statsIterator.next();
+            long rejectionDelta = 0;
+            String name = stats.getName();
```

Comment:
can we use better naming for the variable `name` ?

Reply:
done. renamed it to threadPoolName